.github/workflows/label.yml

```diff
@@ -0,0 +1,12 @@
+name: PR Labeler
+on: [pull_request]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
Hey, Please revert the changes back to what you had before. The Label.yml file was correct and the format for the Labeler.yml file was also correct it just had the wrong labels!

Reply:
You can leave those labels for now just make sure they are referenced in the issue you opened.

Reply:
Reverted https://github.com/giordano/PolynomialRoots.jl/pull/16/commits/cd9bc700e70d9bc4d73b0a0ba952f2cb1ec734bf .


Can I get approved now ?

Regards,
PseudoCodeNerd